conans/client/tools/win.py

```diff
@@ -189,7 +191,7 @@ def vs_installation_path(version, preference=None):
 
     # Try with vswhere()
     try:
-        legacy_products = vswhere(legacy=True)
+        legacy_products = vswhere(legacy=True, prerelease=True)
```

Comment:
I think we don't want to support the prerelease, just to be ready. Feedback @memsharded

Reply:
currently it's impossible to use pre-release versions with conan at all. it would be nice, may be as opt-in (e.g. via config or environment variable), to allow people to test recipes with upcoming VS versions.

Reply:
Up to my knowledge this only affects to automatic profile detection. So putting ``compiler.version=16`` in your profile should work, right? In that case, I'd prefer the autodetection doesn't use pre-releases. Please change it, thanks!

Reply:
removed as discussed